config.sh

```diff
@@ -6,7 +6,7 @@ function pre_build {
     # Runs in the root directory of this repository.
     ADD_CFLAGS=""
     ADD_CONFIG_FLAGS=""
-    GMP_VERSION="6.2.1"  # hasn't changed in 2 years
+    GMP_VERSION="6.3.0"  # hasn't changed in 2 years
```

Comment:
It appears the comment is no longer valid?

```suggestion
    GMP_VERSION="6.3.0"
```